### DISCUSSION TITLE
Acessibility

### DISCUSSION THREAD

**Discussion by avalon60** [TomSchimansky/CustomTkinter#2472] [Ideas]
I hesitate to raise this, but...

Tkinter (and therefore Customtkinter), doesn't directly support accessibility features. I am wondering if this has been considered? I understand that these things can be hand cranked, but it would be much more convenient (less coding effort) if they were part of the framework, allowing users to tab around widgets, visual focus indicators etc.

**Comment by inorton-entrust** [comment]
There seems to be very little info on this, I found one post from the TCL maintainers suggesting that adding screen reader support to the toolkit would be very difficult.

I have seen a couple of cases where developers have tied caret/mouse/keyboard events to their own TTS code for a TK app.

<edit> I have just found https://github.com/python/cpython/issues/145578 which looks like a promising step to adding the tk `accessible` command to core python tkinter
